vpr/src/base/read_route.cpp

```diff
@@ -266,7 +267,7 @@ static void process_nodes(std::ifstream& fp, ClusterNetId inet, const char* file
 
             if (tokens[4] == "to") {
                 format_coordinates(x2, y2, tokens[5], inet, filename, lineno);
-                if (node.xlow() != x || node.xhigh() != x2 || node.yhigh() != y2 || node.ylow() != y) {
+                if (rr_graph.node_xlow(node.id()) != x || rr_graph.node_xhigh(node.id()) != x2 || rr_graph.node_yhigh(node.id()) != y2 || rr_graph.node_ylow(node.id()) != y) {
```

Comment:
Could shorten this code a little by doing:
auto inode = node.id();
and then using inode in the various calls afterward.  I probably have some preference for that, but it isn't critical if you disagree.